tests/testthat/test-record-omega.R

```diff
@@ -533,3 +938,48 @@ test_that("parse_omega_record() aborts on invalid vpair", {
     expect_error(rec$parse(), class = "nmrec_parse_error")
   }
 })
+
+test_that("parse_omega_record() aborts on missing or incomplete scale value", {
+  cases <- list(
+    # no value
+    "$OMEGA SCALE 0.5 0.7",
+    "$OMEGA 0.5 SCALE 0.7",
+    "$OMEGA 0.5 0.7 SCALE",
+    # no closing paren
+    "$OMEGA SCALE(2.0 0.5 0.7",
+    "$OMEGA 0.5 SCALE(2.0 0.7",
+    "$OMEGA 0.5 0.7 SCALE(2.0"
+  )
+
+  for (case in cases) {
+    rec <- record_omega$new("omega", "OMEGA", case)
+    expect_error(rec$parse(), class = "nmrec_parse_error")
+  }
+})
+
+test_that("parse_omega_record() aborts if scale is inside init parens", {
+  cases <- list(
+    # NM-TRAN doesn't support these ("too many nested parentheses").
+    "$OMEGA 1 (fix 2 scale(0.2))",
+    "$OMEGA 1 (2 scale(0.2))"
+  )
+
+  for (case in cases) {
+    rec <- record_omega$new("omega", "OMEGA", case)
+    expect_error(rec$parse(), class = "nmrec_parse_error")
+  }
+})
+
+test_that("parse_omega_record() aborts if scale between init options", {
+  cases <- list(
+    # NM-TRAN permits this, but nmrec doesn't.
```

Comment:
`test-record-unsupported.R` collects other tests for unsupported cases, so I'll move this there.